lifecycle-management/Ahk.Lifecycle.Management/Services/HttpApi/HttpApiService.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections;
+using System.Threading.Tasks;
+
+namespace Ahk.Lifecycle.Management
+{
+    public class HttpApiService : IHttpApiService
```

Comment:
A `HttpService` nem mond arról sokat, mit is csinál ez a szolgáltatás. Sőt, implementációja se túl vastag. Biztos kell ez az osztály?

Reply:
Törölve lett, mert helyette inkább egy IRepository-t fog kapni a HttpTrigger